src/redux/actions/actionTypes.js

```diff
@@ -27,3 +27,8 @@ export const TWITTER_AUTH_FAIL = 'TWITTER_AUTH_FAIL';
 // this will contain actions
 export const CREATE_ARTICLE_SUCCESS = 'CREATE_ARTICLE_SUCCESS';
 export const CREATE_ARTICLE_FAIL = 'CREATE_ARTICLE_FAIL';
+//Reset password actions
+export const ResponseMessage = 'Message';
+export const FormData = 'FormData';
+export const Status = 'Status';
+export const Summary = 'Summary';
```

Comment:
@Anguandia As you might have noticed the team is using a standard naming for action types. Kindly look into these.

Reply:
+1 @minega25

Reply:
@minega25 , @habinezadalvan , actiontype naming streamlined, thanks